Viewer/src/net/relinc/viewer/GUI/SelectCustomDataController.java

```diff
@@ -242,7 +262,7 @@ private void renderStressModifiersVBox(){
 			return;
 		if(stressData instanceof TransmissionPulse){
 			//add 1,2, and 3-wave calculation methods
-			if(getSelectedSample().getCurrentDisplacementDatasubset() instanceof ReflectedPulse)
+			if(getSelectedSample().getResults().get(0).getCurrentDisplacementDatasubset() instanceof ReflectedPulse)
```

Comment:
we assume that all displacements are the same, maybe some flexibility on various ways to compute displacement availible?

Reply:
for now I don't think this will affect behavior. for now, the only thing that changes between the multiple `LoadDisplacementSampleResults` on a sample is `waveType`. So we should be fine choosing the first one here